lib/csv/parser.rb

```diff
@@ -385,23 +420,26 @@ def prepare_regexp
         @row_ends = nil
       end
 
-      @quotes = Regexp.new(escaped_quote_character +
-                           "+".encode(@encoding))
-      if @backslash_quote
-        @quoted_value = Regexp.new("[^".encode(@encoding) +
-                                   escaped_quote_character +
-                                   escaped_backslash_character +
-                                   "]+".encode(@encoding))
-      else
-        @quoted_value = Regexp.new("[^".encode(@encoding) +
-                                   escaped_quote_character +
-                                   "]+".encode(@encoding))
+      if @options[:quote_character]
```

Comment:
Could you use `@quote_character` instead of `@options[:...]`?

Reply:
Yes, Fixed it. https://github.com/ruby/csv/pull/79/commits/817702026739e325e27b735b1ba6473e6d17748c